### PR TITLE
Sirato explorer docs

### PR DESCRIPTION
**Pull request by Adewagold**
## Pull request checklist

Use the following list to make sure your PR fits the Besu documentation quality standard.

### Before creating the pull request

Make sure that:

- [ ] [All commits in this PR are signed off for the DCO](https://wiki.hyperledger.org/display/BESU/DCO).
- [x] You've read the [contribution guidelines](https://wiki.hyperledger.org/display/BESU/Documentation).
- [x] You've [previewed your changes locally](https://wiki.hyperledger.org/display/BESU/Preview+the+documentation).

## Impacted parts

For content changes:

- [ ] Documentation content
- [ ] Documentation page organization

For tool changes:

- [ ] Github Actions workflow
- [ ] Build and QA tools configuration (for example, lint rules or Vale style)
- [ ] MkDocs templates
- [ ] MkDocs configuration
- [ ] Python dependencies
- [ ] Node dependencies and JavaScript
- [ ] Read the Docs configuration
- [ ] GitHub integration

## After creating your PR and tests have finished

Make sure that:

- [ ] You've fixed any issues raised by the tests.
- [ ] You've [previewed your changes on Read the Docs](https://wiki.hyperledger.org/display/BESU/Preview+the+documentation)
  and added a [preview link](#preview).

### CODE REVIEW

File: docs/private-networks/tutorials/sirato-explorer.md
Comment (joshuafernandes) [inline docs/private-networks/tutorials/sirato-explorer.md:44]: Go to the directory `quorum-test-network`, and start the network with:

File: docs/private-networks/tutorials/sirato-explorer.md
Comment (joshuafernandes) [inline docs/private-networks/tutorials/sirato-explorer.md:83]:
```
cd sirato-free/docker-compose 
NODE_ENDPOINT=http://member1besu:8545 docker compose -f docker-compose.yml -f sirato-extensions/docker-compose-quorum-dev-quickstart.yml up
```

Comment (joshuafernandes) [reply docs/private-networks/tutorials/sirato-explorer.md:83]: Added an instruction to go to the folder and updated `docker-compose` to `docker compose` (compose v2 via the plugin)

File: docs/private-networks/tutorials/sirato-explorer.md
Comment (joshuafernandes) [inline docs/private-networks/tutorials/sirato-explorer.md:8]: I'd combine the find out more section at the bottom of the page here, something like:

```
Sirato is an EVM explorer that provides an overview over the whole network, and provides a bunch of additional functionality like block information, contracts metadata, transaction searches etc and an overview can be found [here](https://medium.com/web3labs/epirus-ethereum-saas-blockchain-explorer-d5d961717d15). Sirato supports public and private networks. Sirato free (used here) has some [feature limitations](insert link of what these are ) and you can find out more information about Sirato [here](https://www.web3labs.com/sirato)

This tutorial will walk you through how to setup Sirato Explorer with privacy-enabled Besu network.

```

Update the links but something like that perhaps?

File: docs/private-networks/tutorials/sirato-explorer.md
Comment (joshuafernandes) [inline docs/private-networks/tutorials/sirato-explorer.md:12]: Given the context here is using Sirato with the Quickstart, I'd put this directly in there directly under the block explorer? https://hyperledger-besu--1252.org.readthedocs.build/en/1252/private-networks/tutorials/quickstart/#block-explorer So the context is preserved. 

Add a note to say 'you must have privacy enabled' and then go to line 70 here, 'Start Sirato'

Comment (joshuafernandes) [reply docs/private-networks/tutorials/sirato-explorer.md:12]: Suggest avoiding the repetition and delete lines 12 to 69. Keep focus on Sirato from line 69 onwards?

File: docs/private-networks/tutorials/sirato-explorer.md
Comment (joshuafernandes) [inline docs/private-networks/tutorials/sirato-explorer.md:86]: We've historically avoided things directly on port 80 given people may have something hosted on port 80 and this causes grief, rather pick something else say 25001 or 26000 or something high that won't conflict

File: docs/private-networks/tutorials/sirato-explorer.md
Comment (joshuafernandes) [inline docs/private-networks/tutorials/sirato-explorer.md:93]: Add this sentence right up top after the line indicating it needs privacy

File: docs/private-networks/tutorials/sirato-explorer.md
Comment (joshuafernandes) [inline docs/private-networks/tutorials/sirato-explorer.md:129]: Not sure we need this, if you do need it - highlight what the differences are between free/pro and add a link to Sirato?

File: mkdocs.navigation.yml
Comment (joshuafernandes) [inline mkdocs.navigation.yml:191]: Avoid this as a seperate page and keep the context of the quickstart. Additionally add a line or two in the monitoring page here https://besu.hyperledger.org/en/latest/private-networks/how-to/monitor/?h=monitoring

**Review by joshuafernandes — CHANGES_REQUESTED** [review]
Content looks good, have requested a few changes and can get one of the docs team to review as well and merge

File: docs/private-networks/tutorials/sirato-explorer.md
Comment (joshuafernandes) [inline docs/private-networks/tutorials/sirato-explorer.md:128]: This also needs a section here to stop Sirato and clean up containers like we do in Besu

File: docs/private-networks/tutorials/sirato-explorer.md
Comment (joshuafernandes) [reply docs/private-networks/tutorials/sirato-explorer.md:83]: Does this need to remain in foreground or can we add the `-d` flag to compose to put it in the background?

Comment (Adewagold) [reply docs/private-networks/tutorials/sirato-explorer.md]: Yes. We will add a script to start and stop explorer, and this will run the service in background.

File: docs/private-networks/tutorials/sirato-explorer.md
Comment (Adewagold) [reply docs/private-networks/tutorials/sirato-explorer.md]: This will be flexible to start sirato explorer with besu on port 26000, while the default port to run the explorer as an independent service will use port 80.

Comment (Adewagold) [reply docs/private-networks/tutorials/sirato-explorer.md:75]: Besu can now be started using by using the PORT variable when running docker-compose 
```
cd docker-compose
NODE_ENDPOINT=member1besu PORT=26000 docker-compose -f docker-compose.yml -f sirato-extensions/docker-compose-quorum-dev-quickstart.yml up
```

Comment (joshuafernandes) [reply docs/private-networks/tutorials/sirato-explorer.md:75]: 👍 
Thanks for sorting this out

File: docs/private-networks/tutorials/quickstart.md
Comment (joshuafernandes) [inline docs/private-networks/tutorials/quickstart.md:116]: Just a very minor tweak here, can we change to `[Sirato](https://www.web3labs.com/sirato) is an EVM.....  and end the sentence with the feature limitations link` This brings the focus back to Sirato's right from the beginning which is easier to follow, rather than saying it does all these features and it has some limitations, etc and this is who we are.

File: docs/private-networks/tutorials/quickstart.md
Comment (joshuafernandes) [inline docs/private-networks/tutorials/quickstart.md:182]: Add `From sirato-free repo, run the following command` just to be clear and avoid support questions

**Review by joshuafernandes — COMMENTED** [review]
Couple of minor tweaks to this requested, but looks good. Thankyou @adewagold. Once its out of draft mode and the checks are sorted, one of the docs-maintainers should approve and can merge

**Comment by alexandratran** [comment]
I migrated this content over to #1265, and edited the content to fit documentation standards. This was the easiest way to deal with the DCO errors on this PR, since there are many commits from multiple contributors without sign-offs. Please follow Hyperledger's [DCO requirement](https://wiki.hyperledger.org/display/BESU/DCO) for future contributions to this repository.

**Pull request closed without merging**
